io.github.zer0x.qurancompanion.yml

```diff
@@ -0,0 +1,43 @@
+app-id: io.github.zer0x.qurancompanion
+runtime: org.kde.Platform
+runtime-version: '6.5'
+sdk: org.kde.Sdk
+command: quran-companion
+finish-args:
+  - --share=ipc
+  - --share=network
+  - --socket=x11
+  - --socket=wayland
+  - --socket=session-bus
```

Comment:
session-bus is NOT allowed. You need to determine which name you need instead.

Reply:
I need system tray support, which service name do I need to use?